src/index.js

```diff
@@ -1 +1,22 @@
-console.log('Works!');
\ No newline at end of file
+import Project from './projects';
+import createStorage from './storage';
+import createProject from './createProject';
+
+const botonProject = document.getElementById('botonProject');
+
+function initTodo() {
+  botonProject.addEventListener('click', () => {
+    const nameProject = document.getElementById('projectName').value;
+    const project = new Project(nameProject);
+
+    if (nameProject === '') {
+      alert('Project has to have a name!');
```

Comment:
Kindly remove `alert`

Reply:
Thank you very much for the review, have a nice day. 👍🏼